routes/api.php

```diff
@@ -46,6 +46,7 @@
         Route::post('projects/{project}/remove-share', 'ProjectController@removeShare');
         Route::get('projects/recent', 'ProjectController@recent');
         Route::get('projects/default', 'ProjectController@default');
+        Route::get('projects/detailed', 'ProjectController@listDetailed');
```

Comment:
This can be 
/projects/detail

And function in projects can just be 'detail()'.

Reply:
done